code/modules/mining/equipment_locker.dm

```diff
@@ -15,51 +17,116 @@
 		access_chemistry,
 		access_bar,
 		access_research,
-		access_ce,
-		access_virology
+		access_ce
```

Comment:
Realize this is still WIP, but I believe they handle this with an all new access type on TG "access_mineral_storeroom"